snap/snapcraft.yaml

```diff
@@ -63,42 +50,28 @@ parts:
     organize:
       '*': bin/
 
-  # Check out the tagged release revision if it isn't promoted to the stable channel
-  # https://forum.snapcraft.io/t/selective-checkout-check-out-the-tagged-release-revision-if-it-isnt-promoted-to-the-stable-channel/10617
-  selective-checkout:
-    source: https://github.com/Lin-Buo-Ren/selective-checkout.git
-    source-tag: v2.0.0
-    plugin: dump
-    build-packages:
-    # Scriptlet dependencies
-    - curl
-    - git
-    - jq
-    - sed
-    stage:
-    - scriptlets/selective-checkout
-    prime:
-    - -*
-
   gallery-dl:
-    after:
-    - selective-checkout
-
     source: .
     override-pull: |
-      snapcraftctl pull
-      $SNAPCRAFT_STAGE/scriptlets/selective-checkout
+      craftctl default
+      ${CRAFT_PROJECT_DIR}/snap/local/scriptlets/selective-checkout
 
     plugin: python
     build-packages:
     - make
+
+    # selective-checkout scriptlet dependencies
+    - curl
+    - git
+    - jq
+    - sed
     python-packages:
     - youtube_dl
```

Comment:
Could this be updated to `yt-dlp` or `yt-dlp[default]`?

Reply:
The requested change is made in https://github.com/mikf/gallery-dl/pull/7865 .